idaes/ui/fsvis/templates/index.html

```diff
@@ -6,9 +6,13 @@
   <link rel="icon" href="idaes-logo.ico" type="image/x-icon">
   <link href="https://fonts.googleapis.com/css2?family=Raleway&display=swap" rel="stylesheet">
   <script src="https://kit.fontawesome.com/b27a429048.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
```

Comment:
You are adding all the bootstrap files locally.. but then loading them from a CDN too? I suggest we stick with the local data, assuming collaborators may be behind firewalls etc.

Reply:
Good catch!